### PR TITLE
add main function

### PR DESCRIPTION
**Pull request by LYZ1125**
it seems that you forget to adding the main function.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -8,3 +8,9 @@ def edit_distance(str1, str2):
                 d = 1
             edit[i][j] = min(edit[i - 1][j] + 1, edit[i][j - 1] + 1, edit[i - 1][j - 1] + d)
     return edit[len(str1)][len(str2)]
+# can I commit like this ?
+if __name__ == '__main__':
```
Comment (LYZ1125) [inline test.py:12]: is it OK?

Comment (Ying091909) [inline test.py:12]: you are right

File: test.py [hunk 2]
```diff
@@ -8,3 +8,9 @@ def edit_distance(str1, str2):
...
+    str1 = "li ying da ben dan";
+    str2 = "liing daben dan";
+    result = edit_distance(str1, str2)
+    print(result)
```
Comment (Ying091909) [inline test.py:16]: good

File: test.py (hunk 1 above)
Comment (LYZ1125) [reply test.py:12]: is this a new review?

Comment (LYZ1125) [reply test.py:12]: what's mean of pending? why do i have this label?

File: test.py (hunk 2 above)
Comment (LYZ1125) [inline test.py:16]: if I comment here.....

**Review by Ying091909 — COMMENTED** [review]
are you ok?

File: test.py (hunk 1 above)
Comment (Ying091909) [reply test.py:12]: ok

**Pull request merged**